runtime/jcl/common/sun_misc_Perf.c

```diff
@@ -91,6 +91,15 @@ Java_sun_misc_Perf_highResFrequency(JNIEnv *env, jobject perf)
 	return j9time_hires_frequency();
 }
 
+#if JAVA_SPEC_VERSION >= 19
+/* private native ByteBuffer attach0(String user, int lvmid) throws IllegalArgumentException, IOException; */
+jobject JNICALL
+Java_jdk_internal_perf_Perf_attach(JNIEnv *env, jobject perf, jstring user, jint lvmid)
+{
```

Comment:
should assert as well

Reply:
I don't see the assertion Tobi suggested.

Reply:
no assertion has been added to follow the existing behavior of `Java_sun_misc_Perf_attach`

Reply:
I think both should assert: callers expect a non-null value to be returned (which neither function does).
@tajila Opinons?

Reply:
Ya, I think we should assert those. We currently never call this, but if we do in the future its better to assert than  fail somewhere else.